electronics/details/details_flashing.md

```diff
@@ -0,0 +1,23 @@
+# Details Flashing
+
+This document describes how to flash the firmware on the TI LaunchPad Development Kit.
+
+## Step 1 - Remove jumpers
+
+After unpacking the LaunchPad, 
+* Remove the jumpers JP1, JP2, JP4, JP5
+* Enable the jumper JP6
+
+## Step 2 - Install software to flash the firmware
+
+To be written.
```

Comment:
I would prefer a "TODO" here as it stands out more so it is less easy for us to miss it.